README.md

```diff
@@ -228,26 +224,37 @@ GTMSessionFetcher *fetcher = [fetcherService fetcherWithURL:userinfoEndpoint];
 
 ### Saving to the Keychain
 
-You can easily save `GTMAppAuthFetcherAuthorization` instances to the Keychain using
-the included `GTMAppAuthFetcherAuthorization+Keychain` category.
+You can easily save `GTMAuthSession` instances to the Keychain using the `GTMKeychainStore` class.
```

Comment:
Is it worth mentioning that we provide that as a convenience? I.e., that they can make their own conformance to `AuthSessionStore` to persist `AuthSession`s.

Reply:
Added an "Implementing Your Own Storage" section below.